docs/concepts/storage/volumes.md

```diff
@@ -854,6 +854,58 @@ volume plugin path on each node. This is an alpha feature and may change in futu
 More details can be found [here](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md).
 
 
+## Mount propagation
+
+**Note:** Mount propagation is alpha feature in Kubernetes 1.8 and may be
+redesigned or even removed in future releases.
+{: .note}
+
+Mount propagation allows for sharing volumes mounted by a Container in a Pod to
+other Containers in the Pod or even to other Pods on the same node.
+
+If MountPropagation feature is disabled, all volume mounts in pods are not
+propagated (i.e. run with `private` mount propagation as described in
+[Linux kernel documentation](https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt))
+
+To enable this feature, the `MountPropagation=true` option has to be specified
+in the `--feature-gates` command line option. When enabled, `volumeMounts` field
+in Container gets a new field `mountPropagation`. Its values are:
+
```

Comment:
Mount propagation allows for sharing volumes mounted by a Container to
other Containers in the same Pod, or even to other Pods on the same node.

If the MountPropagation feature is disabled, volume mounts in pods are not propagated.
That is, Containers run with `private` mount propagation as described in the
[Linux kernel documentation](https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt).

To enable this feature, specify `MountPropagation=true` in the
`--feature-gates` command line option. When enabled, the `volumeMounts` field
of a Container has a new `mountPropagation` subfield. Its values are: